custom.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
+    import React = require('react');
```

Comment:
usually it is a good idea to keep your imports in top-level in declaration files